### DISCUSSION TITLE
Error while uploading XML on ODK aggregate.

### DISCUSSION THREAD

**Discussion by SaketSinha** [Samagra-Development/SamagraX-community#22] [Troubleshooting and Clarifications]
While trying to upload an XML file on the Ashish ji (NIC haryana) is facing this issue:

![WhatsApp Image 2021-04-02 at 16 49 38](https://user-images.githubusercontent.com/25716415/113411666-0cadd200-93d4-11eb-852e-f4e3060e67ee.jpeg)

The ODK Aggregate being used is:
URL = https://agg.staging.saksham.samagra.io/
Username = administrator 

cc- @aayushkedia95 @ChakshuGautam

**Comment by SaketSinha** [comment]
@ashupandith - Please share the related Excel and XML files for debugging.

**Reply by ashupandith** [reply]
@SaketSinha 
[Base XLS Form.xlsx](https://github.com/Samagra-Development/SamagraX-community/files/6262231/Base.XLS.Form.xlsx)

Unable to send the XML file . So just pased over here 

<?xml version="1.0"?><h:html xmlns="http://www.w3.org/2002/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:odk="http://www.opendatakit.org/xforms" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema"><h:head><h:title>Base XLS Form</h:title><model odk:xforms-version="1.0.0"><itext><translation lang="English"><text id="/data/group_faq:label"><value>Saral Kendra details</value></text><text id="/data/group_faq/name:label"><value>what is your name?</value></text><text id="/data/group_saraldetail:label"><value>Saral Kendra details</value></text><text id="/data/group_saraldetail/district_saralkendra:label"><value>District</value></text><text id="/data/group_saraldetail/district_saralkendra/chamba:label"><value>chamba</value></text><text id="/data/group_saraldetail/district_saralkendra/una:label"><value>una</value></text><text id="/data/group_saraldetail/district_saralkendra/hamirpur:label"><value>hamirpur</value></text><text id="/data/group_saraldetail/district_saralkendra/panchkula:label"><value>panchkula</value></text><text id="/data/group_saraldetail/district_saralkendra/ambala:label"><value>ambala</value></text><text id="/data/group_saraldetail/district_saralkendra/shimla:label"><value>shimla</value></text><text id="/data/group_saraldetail/district_saralkendra/sirmaur:label"><value>sirmaur</value></text><text id="/data/group_saraldetail/district_saralkendra/chamba -U:label"><value>chamba -U</value></text><text id="/data/group_saraldetail/district_saralkendra/kullu:label"><value>kullu</value></text><text id="/data/group_saraldetail/district_saralkendra/mandi:label"><value>mandi</value></text><text id="/data/group_saraldetail/district_saralkendra/kinnaur:label"><value>kinnaur</value></text><text id="/data/group_saraldetail/district_saralkendra/solan:label"><value>solan</value></text><text id="/data/group_saraldetail/district_saralkendra/kangra:label"><value>kangra</value></text><text id="/data/group_saraldetail/district_saralkendra/bilaspur:label"><value>bilaspur</value></text><text id="/data/group_saraldetail/block_saralkendra:label"><value>block</value></text><text id="opt_district_saralkendra-0"><value>chamba</value></text><text id="opt_district_saralkendra-1"><value>una</value></text><text id="opt_district_saralkendra-2"><value>hamirpur</value></text><text id="opt_district_saralkendra-3"><value>panchkula</value></text><text id="opt_district_saralkendra-4"><value>ambala</value></text><text id="opt_district_saralkendra-5"><value>shimla</value></text><text id="opt_district_saralkendra-6"><value>sirmaur</value></text><text id="opt_district_saralkendra-7"><value>chamba -U</value></text><text id="opt_district_saralkendra-8"><value>kullu</value></text><text id="opt_district_saralkendra-9"><value>mandi</value></text><text id="opt_district_saralkendra-10"><value>kinnaur</value></text><text id="opt_district_saralkendra-11"><value>solan</value></text><text id="opt_district_saralkendra-12"><value>kangra</value></text><text id="opt_district_saralkendra-13"><value>bilaspur</value></text><text id="opt_block_saralkendra-0"><value>badoli</value></text><text id="opt_block_saralkendra-1"><value>basal</value></text><text id="opt_block_saralkendra-2"><value>turi</value></text><text id="opt_block_saralkendra-3"><value>amb</value></text><text id="opt_block_saralkendra-4"><value>badsala</value></text><text id="opt_block_saralkendra-5"><value>sammor</value></text><text id="opt_block_saralkendra-6"><value>arnyala</value></text><text id="opt_block_saralkendra-7"><value>raipur</value></text><text id="opt_block_saralkendra-8"><value>mathtapur</value></text><text id="opt_block_saralkendra-9"><value>dehla</value></text><text id="opt_block_saralkendra-10"><value>badala</value></text><text id="opt_block_saralkendra-11"><value>chatada</value></text></translation></itext><instance><data id="Base XLS Form"><group_faq><name/></group_faq><group_saraldetail><district_saralkendra/><block_saralkendra/></group_saraldetail><meta><instanceID/></meta></data></instance><instance id="opt_district_saralkendra"><root><item><itextId>opt_district_saralkendra-0</itextId><name>chamba</name></item><item><itextId>opt_district_saralkendra-1</itextId><name>una</name></item><item><itextId>opt_district_saralkendra-2</itextId><name>hamirpur</name></item><item><itextId>opt_district_saralkendra-3</itextId><name>panchkula</name></item><item><itextId>opt_district_saralkendra-4</itextId><name>ambala</name></item><item><itextId>opt_district_saralkendra-5</itextId><name>shimla</name></item><item><itextId>opt_district_saralkendra-6</itextId><name>sirmaur</name></item><item><itextId>opt_district_saralkendra-7</itextId><name>chamba -U</name></item><item><itextId>opt_district_saralkendra-8</itextId><name>kullu</name></item><item><itextId>opt_district_saralkendra-9</itextId><name>mandi</name></item><item><itextId>opt_district_saralkendra-10</itextId><name>kinnaur</name></item><item><itextId>opt_district_saralkendra-11</itextId><name>solan</name></item><item><itextId>opt_district_saralkendra-12</itextId><name>kangra</name></item><item><itextId>opt_district_saralkendra-13</itextId><name>bilaspur</name></item></root></instance><instance id="opt_block_saralkendra"><root><item><itextId>opt_block_saralkendra-0</itextId><cf>una</cf><name>badoli</name></item><item><itextId>opt_block_saralkendra-1</itextId><cf>chamba</cf><name>basal</name></item><item><itextId>opt_block_saralkendra-2</itextId><cf>hamirpur</cf><name>turi</name></item><item><itextId>opt_block_saralkendra-3</itextId><cf>mandi</cf><name>amb</name></item><item><itextId>opt_block_saralkendra-4</itextId><cf>una</cf><name>badsala</name></item><item><itextId>opt_block_saralkendra-5</itextId><cf>mandi</cf><name>sammor</name></item><item><itextId>opt_block_saralkendra-6</itextId><cf>hamirpur</cf><name>arnyala</name></item><item><itextId>opt_block_saralkendra-7</itextId><cf>chamba</cf><name>raipur</name></item><item><itextId>opt_block_saralkendra-8</itextId><cf>una</cf><name>mathtapur</name></item><item><itextId>opt_block_saralkendra-9</itextId><cf>una</cf><name>dehla</name></item><item><itextId>opt_block_saralkendra-10</itextId><cf>chamba</cf><name>badala</name></item><item><itextId>opt_block_saralkendra-11</itextId><cf>hamirpur</cf><name>chatada</name></item></root></instance><bind nodeset="/data/group_faq/name" required="true()" type="string"/><bind nodeset="/data/group_saraldetail/district_saralkendra" required="true()" type="string"/><bind nodeset="/data/group_saraldetail/block_saralkendra" required="true()" type="string"/><bind jr:preload="uid" nodeset="/data/meta/instanceID" readonly="true()" type="string"/></model></h:head><h:body><group appearance="field-list" ref="/data/group_faq"><label ref="jr:itext('/data/group_faq:label')"/><input ref="/data/group_faq/name"><label ref="jr:itext('/data/group_faq/name:label')"/></input></group><group appearance="field-list" ref="/data/group_saraldetail"><label ref="jr:itext('/data/group_saraldetail:label')"/><select1 ref="/data/group_saraldetail/district_saralkendra"><label ref="jr:itext('/data/group_saraldetail/district_saralkendra:label')"/><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/chamba:label')"/><value>chamba</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/una:label')"/><value>una</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/hamirpur:label')"/><value>hamirpur</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/panchkula:label')"/><value>panchkula</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/ambala:label')"/><value>ambala</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/shimla:label')"/><value>shimla</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/sirmaur:label')"/><value>sirmaur</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/chamba -U:label')"/><value>chamba -U</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/kullu:label')"/><value>kullu</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/mandi:label')"/><value>mandi</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/kinnaur:label')"/><value>kinnaur</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/solan:label')"/><value>solan</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/kangra:label')"/><value>kangra</value></item><item><label ref="jr:itext('/data/group_saraldetail/district_saralkendra/bilaspur:label')"/><value>bilaspur</value></item></select1><select1 ref="/data/group_saraldetail/block_saralkendra"><label ref="jr:itext('/data/group_saraldetail/block_saralkendra:label')"/><itemset nodeset="instance('opt_block_saralkendra')/root/item[cf= /data/group_saraldetail/district_saralkendra ]"><value ref="name"/><label ref="jr:itext(itextId)"/></itemset></select1></group></h:body></h:html>

**Reply by ashupandith** [reply]
@SaketSinha 
[xml.txt](https://github.com/Samagra-Development/SamagraX-community/files/6262238/xml.txt)

Sending the XML file in a txt file

**Reply by aayushkedia95** [reply]
@ashupandith, please add form name, form id and version in the settings tab. Adding corrected .xlsx file here. Please use this for reference.   
[Base.XLS.Form.xlsx](https://github.com/Samagra-Development/SamagraX-community/files/6264550/Base.XLS.Form.xlsx)
